src/core/Toast/Toast.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { Component, forwardRef, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { baseStyles } from './Toast.baseStyles';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../theme';
+import { Icon } from '../Icon/Icon';
+import { Heading } from '../Heading/Heading';
+import { HtmlDiv, HtmlDivWithRef } from '../../reset';
+import { hLevels } from '../../reset/HtmlH/HtmlH';
+
+export interface ToastProps {
+  /** Set aria-live mode for the Toast text content and label.
+   * @default 'polite'
+   */
+  ariaLiveMode?: 'polite' | 'assertive' | 'off';
+  /** Heading for the Toast */
+  headingText?: string;
+  /** Text content of the Toast. */
```

Comment:
Should we change this to "Main content --" instead of "Text content --" since the type is not limited to text?

Reply:
Good point! changed:
/** Text content of the Toast. */
to:
/** Main content of the Toast. */